packages/xml-views-validation/api.d.ts

```diff
@@ -4,6 +4,7 @@ import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
 export function validateXMLView(opts: {
   model: UI5SemanticModel;
   xmlView: XMLDocument;
+  flexEnabled: boolean;
```

Comment:
make flexEnabled optional (default value at runtime) so we will avoid a **breaking change** in the API